editor/src/core/shared/multiplayer.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { multiplayerInitialsFromName } from './multiplayer'
+
+describe('multiplayer', () => {
+  describe('multiplayerInitialsFromName', () => {
+    const tests: {
+      name: string
+      input: string
+      want: string
+    }[] = [
+      {
+        name: 'regular name',
+        input: 'Foo Bar',
+        want: 'FB',
+      },
+      {
+        name: 'name with many words',
+        input: 'Foo Bar Baz',
+        want: 'FB',
+      },
+      {
+        name: 'single word',
+        input: 'Bar',
+        want: 'BA',
+      },
+      {
+        name: 'a very short name',
+        input: 'F',
+        want: 'FX',
+      },
+      {
+        name: 'no name at all',
+        input: '',
+        want: 'XX',
+      },
+    ]
+    tests.forEach((test, index) => {
+      it(`${index}/${tests.length + 1} ${test.name}`, async () => {
```

Comment:
I believe werre trying to move away from generating the test name like this because it's not so easy to find a test when it fails.

Reply:
yes please! ideally we should be writing these test cases in the shape of:

```typescript
      it('a very short name',() => {
        runTest({
          input: 'F',
          want: 'FX',
       })
     })
```